content/games/submissions/arm_bar/breaking_the_grip/_index.md

```diff
@@ -0,0 +1,20 @@
++++
+archetype = "game"
+title = "Break their Grip"
+tags = ["submissions", "arm bar", "slime mold originals"]
+weight = 1
```

Comment:
Could we set this to 2 so that it sorts after the existing arm bar game in the nav menu?